net/routing/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       routing
+PLUGIN_VERSION=    0.0.1
```

Comment:
fair enough. it's up to you. :) I just want to share the relevant info: users have voiced concerns about long version numbers early in OPNsense 15.1.x and we also have port revisions to deal with smaller changes here (PORTREVISION=1 etc.)